src/core/evaluator.js

```diff
@@ -999,6 +1016,14 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
                   operatorList.addOp(OPS.setFont, [loadedName, fontSize]);
                 }));
               return;
+            case OPS.beginText:
+              parsingText = true;
+              args = null;
+              break;
+            case OPS.endText:
+              parsingText = false;
+              args = null;
```

Comment:
Why does `args` need to be set to `null` here and above?

Reply:
It's a "better safe than sorry" approach really, since neither of the `BT`/`ET` operators *should* ever have parameters. However, considering the kind of checks that happen in the `default` branch, just always resetting the `args` seemed easiest; see https://github.com/mozilla/pdf.js/blob/f87dc42780d9aaefe3b3c01c1127d88d05ac5763/src/core/evaluator.js#L1180-L1194

Reply:
Right, but now that we're actually handling these cases explicitly we don't enter the `default` branch for them anymore, so unless I'm missing something here the `args` are not actually used anywhere.

Reply:
> [...] so unless I'm missing something here the `args` are not actually used anywhere.

That's absolutely correct, but I was a bit worried that https://github.com/mozilla/pdf.js/blob/f87dc42780d9aaefe3b3c01c1127d88d05ac5763/src/core/evaluator.js#L2991-L3004 could *somehow* cause a `BT`/`ET` operator to end up with an `args` array.
Given that `getOperatorList` always calls `EvaluatorPreprocessor.read` in as outlined in https://github.com/mozilla/pdf.js/blob/f87dc42780d9aaefe3b3c01c1127d88d05ac5763/src/core/evaluator.js#L2961-L2967 that worry seems to be unfounded though, so I'll simply remove those lines again.

Really good point!